### DISCUSSION TITLE
Welcome to the accessibility feedback category 👋

### DISCUSSION THREAD

**Discussion by benjiallen** [community/community#32773] [Accessibility]
### Welcome!

This category is a place for our community to discuss and provide feedback on the accessibility of GitHub products.

#### What is accessibility?

Accessibility is about creating highly usable products for all of our customers, no matter their physical or cognitive ability. If you're new or curious about this topic, you might enjoy the [W3C's Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/).

If you have a suggestion for improvement, a feature request, or just want to share your thoughts, let us know in the discussions here. We love hearing from our customers with disabilities!

### Helpful links
* Check out [our first Welcome to the Accessibility Feedback Category](https://github.com/orgs/community/discussions/16996).

* Check out the [repository README](https://github.com/github/feedback#github-public-feedback-discussions) and our [code of conduct](https://github.com/github/feedback/blob/main/CODE_OF_CONDUCT.md) to learn more about what to expect and how to engage.

* For general help, please head to the [GitHub docs](https://docs.github.com/en)

**Comment by ghostinhershell** [comment]
Thanks for the post and the helpful links!

**Comment by RichCaloggero** [comment]
Here are a couple of random thoughts on how the github experience might be improved for screen reader users.

## Github Issue Lists

It would be helpful for screen reader users to have a table view (using HTML data table markup, not ARIA) presenting the issue list. Semantically, it is a table, so use semantic HTML to present it.

If this wouldn't be helpful for visually oriented users, then offer a way to switch views, and even better, a way to save your view preference in your account.

## Command pallet

Just found this today, and while a great idea, it does not help screen reader users at all because the suggestions aren't navicable and the screen reader isn't made aware when they appear.

There are many posts around the web on how to build completion widgets (sometimes called comboboxes). 

Instead, might want to consider HTML datalist markup. This provides a way to build completion lists and attach them to `input` elements in markup (no javascript or aria needed)!  Might also use a live region to tell the screen reader when suggestions appear (i.e. 5 repositories match, 5 users match, etc).

**Reply by ghostinhershell** [reply]
Hey @RichCaloggero, thanks for the great feedback! Do you mind creating a Discussion post in the Accessibility category with this information as well?

**Comment by letusskate** [comment]
I really want to export github project to csv or xlsx to manage my project more conveniently.
